.github/actions/dump-disk-info/action.yaml

```diff
@@ -0,0 +1,16 @@
+name: Dump disk space info
+description: Help debug running out of disk space on github CI
+inputs:
+  platform:
+    description: "Platform name"
+    required: true
+runs:
+  using: "composite"
```

Comment:
I would like to only run this if platform is linux. I haven't looked up the github action syntax for that just yet

Reply:
maybe something like:

```
if: ${{ runner.os != 'macOS' }
```

or 
```
if: ${{ runner.os != 'Linux' }
```

on the action (https://docs.github.com/en/actions/learn-github-actions/contexts#runner-context)